dhcpv6/dhcpv6message.go

```diff
@@ -101,6 +102,12 @@ func NewAdvertiseFromSolicit(sol *Message, modifiers ...Modifier) (*Message, err
 		return nil, errors.New("Client ID cannot be nil in SOLICIT when building ADVERTISE")
 	}
 	adv.AddOption(cid)
+	// add IA_NA
```

Comment:
I would leave this to the server implementation. Not having a IA_NA is not a reason to hard fail, client may be requesting IA_PD, etc.

Reply:
good point. I will remove this

Reply:
done